lib/global-styles.php

```diff
@@ -71,34 +71,22 @@ function_exists( 'gutenberg_is_edit_site_page' ) &&
 		}
 
 		$new_global_styles = array();
-		$new_presets       = array(
-			array(
-				'css'                     => 'variables',
```

Comment:
We no longer pass the CSS variables via the block editor settings and tell the editor not to wrap them. Instead, we enqueue them via the `enqueue_block_editor_assets` filter.